src/database/migrations/20231211041333_CreateApiTable.ts

```diff
@@ -0,0 +1,32 @@
+import { Knex } from 'knex';
+
+import { ApiRole } from '../../api/entities/ApiRole';
+import { Participant } from '../../api/entities/Participant';
+import { getSiteList } from '../../api/services/adminServiceClient';
+import { SiteDTO } from '../../api/services/adminServiceHelpers';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('apiRoles', (table) => {
+    table.increments('id').primary();
+    table.string('roleName', 100).notNullable();
+  });
+
+  // Add default roles
+  const defaultRoles = ['MAPPER', 'GENERATOR', 'ID_READER', 'SHARER', 'OPTOUT'];
+  await knex('apiRoles').insert(
+    defaultRoles.map((role) => {
+      return { roleName: role };
+    })
+  );
+
+  await knex.schema.createTable('participantsToRoles', (table) => {
```

Comment:
Perhaps `participantsToApiRoles` instead of `participantsToRoles`? Since we may implement roles (by user) in the future - this would avoid future confusion.